urllib3/response.py

```diff
@@ -422,6 +422,18 @@ def _handle_chunk(self, amt):
             self.chunk_left = None
         return returned_chunk
 
+    @staticmethod
```

Comment:
I thought about moving it to utils/connection or so, where the other platform specific stuff lives.